src/accessibility.cs

```diff
@@ -1,9 +1,96 @@
 using System;
+using CoreGraphics;
 using Foundation;
 using ObjCRuntime;
 
 namespace Accessibility {
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXCategoricalDataAxisDescriptor : AXDataAxisDescriptor
+	{
+		[Export ("categoryOrder", ArgumentSemantic.Copy)]
+		string[] CategoryOrder { get; set; }
+
+		[Export ("initWithTitle:categoryOrder:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string title, string[] categoryOrder);
+
+		[Export ("initWithAttributedTitle:categoryOrder:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSAttributedString attributedTitle, string[] categoryOrder);
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
```

Comment:
no `[BaseType]` unless there's a `[Model]`

Reply:
By "unless there's a [Model]", do you mean unless the class inherits from another class, or unless the class literally has the attribute `[Model]`? And how do we know when the tag is needed?

Reply:
He means the attribute, since Model will create an instance.

Reply:
The `[Model]` attribute, you only add `[Model]`, `[BaseType]` and `[Protocol]` to types with a suffix `Delegate` or `DataSource` so `FooDelegate` and `FooDataSource` will need the 3 of them any other `@protocol` just needs the `[Protocol]` attribute.

Reply:
gotcha, thanks for clarifying!

Reply:
Should I also remove the attribute from existing types in the file, or just let them be?

Reply:
I may be wrong, but I think we try to leave things already there alone unless it causes an issue

Reply:
Do not remove existing attributes since it will be a breaking change